perceval/backends/gerrit.py

```diff
@@ -258,24 +323,38 @@ def next_retrieve_group_item(self, last_item=None, entry=None):
                 next_item = 0
             else:
                 next_item = last_item
+        elif gerrit_version[0] == 2 and gerrit_version == 9:
+            # https://groups.google.com/forum/#!topic/repo-discuss/yQgRR5hlS3E
+            raise RuntimeExecption("Gerrit 2.9 does not support pagination")
```

Comment:
This exception does not exist and you misspelled it. Change it by the generic `BackendError`.